spec/dummy/app/views/home/index.html.haml

```diff
@@ -6,8 +6,6 @@
     %title= page_title # page_title is defined in Helpers
     = csrf_meta_tags
     = csp_meta_tag
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT" crossorigin="anonymous">
```

Comment:
After this, the dummy app doesn't include any bootstrap styles. It probably should